sdk/data_tables/src/transaction.rs

```diff
@@ -40,7 +40,7 @@ impl Transaction {
         for transaction_operation in self.transaction_operations.iter() {
             s.push_str(&format!("--changeset_{}\nContent-Type: application/http\nContent-Transfer-Encoding: binary\n\n", self.change_set_uuid.hyphenated()));
             s.push_str(&format!(
-                "{} {} HTTP/1.1\n",
+                "{:?} {} HTTP/1.1\n",
```

Comment:
Not sure I like that `method` doesn't have a display. The string representations of the HTTP methods are cononical no?

Reply:
> The string representations of the HTTP methods are cononical no?

Correct. As per [RFC 9110, 9: Methods](https://www.rfc-editor.org/rfc/rfc9110#name-methods):

> The method token is case-sensitive because it might be used as a gateway to object-based systems with case-sensitive method names. By convention, standardized methods are defined in all-uppercase US-ASCII letters.